starter-code/index.js

```diff
@@ -1,10 +1,45 @@
-var SortedList = function() {};
-
-SortedList.prototype.add     = function(item) {}
-SortedList.prototype.get     = function(pos) {}
-SortedList.prototype.max     = function() {}
-SortedList.prototype.min     = function() {}
-SortedList.prototype.average = function() {}
-SortedList.prototype.sum     = function() {}
+var SortedList = function() {
+  this.items = [],
+  this.length = this.items.length
+};
+
+SortedList.prototype.add     = function(item) {
+this.items.push(item)
+this.items.sort(function(a,b){
+  return a - b
+})
+this.length ++;
+}
+SortedList.prototype.get     = function(pos) {
+  if(pos < 1 || pos > this.length){
+    throw Error ("OutOfBounds");
```

Comment:
You are not supposed to write any "if" or "throw Error" .... when we look at tests, if you write exactly what the test is "testing" then your not really testing your code but rather just fooling the test into thinking you passed it. 

This was the correct way for this one:
```js
SortedList.prototype.get = function(pos) {
  return this.list[pos - 1];
};
```